modules/guides/examples/javascript-udfs/execute-javascript-function.sh

```diff
@@ -0,0 +1,3 @@
+curl -v http://localhost:8093/query/service \
+  -u Administrator:password \
+  -d 'statement=EXECUTE FUNCTION GetBusinessDays("02/14/2022", "04/16/2022")'
```

Comment:
Best practice: semicolon at end of N1QL statement

```suggestion
  -d 'statement=EXECUTE FUNCTION GetBusinessDays("02/14/2022", "04/16/2022");'
```